README.md

```diff
@@ -1,2 +1,27 @@
 # keepTrack
+
 premises access time
+
+
+# Setup
+
+```bash
+npm install
+
+# Start frontend
+ng serve
+
+# Start backend
+npm start
+```
+
+## Database
+
+```bash
+docker run --name keepTrack -p 6000:5432 -e POSTGRES_PASSWORD=duckTracky -e POSTGRES_DB=keepTrack -d postgres
+
+# Manually create database "keepTrack"
```

Comment:
TODO: Find a fix for this

+ Add migration to add first user?
+ Add migration to change data types of Logs.userId and Logitem.logId to integer